pipelines/src/test/groovy/VersionInfoTest.groovy

```diff
@@ -174,7 +175,8 @@ class VersionInfoTest {
         Assertions.assertEquals("11.0.9.1+1", parsed.version)
         Assertions.assertNull(parsed.opt)
         Assertions.assertEquals(2, parsed.adopt_build_number)
-        Assertions.assertEquals("11.0.9+11.2", parsed.semver)
+        Assertions.assertEquals("11.0.9+101.2", parsed.semver)
+        Assertions.assertEquals("11.0.9.101", parsed.msi_product_version)
```

Comment:
Do we need to cover the .0 or .1 suffix cases here?

Reply:
Added no AdoptNumber test